org.osgi.service.jakartaws/src/org/osgi/service/jakartaws/runtime/dto/HandlerDTO.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0 
+ *******************************************************************************/
+package org.osgi.service.jakartaws.runtime.dto;
+
+import org.osgi.dto.DTO;
+import org.osgi.framework.dto.ServiceReferenceDTO;
+
+/**
+ * Represents a handler currently known to the webservice runtime
+ */
+public class HandlerDTO extends DTO {
+	/**
+	 * The service reference of the handler, is never {@code null}.
+	 */
+	public ServiceReferenceDTO serviceReference;
+
+	/**
+	 * Returns the representations of the Web Services endpoints this handler is
+	 * currently bound to, the returned array may be empty.
+	 */
+	public EndpointDTO[]		boundEndpoints;
```

Comment:
As described in https://github.com/osgi/osgi/pull/659/files#r1697146194 - this causes a cycle in the DTOs which is forbidden.